Add theme option

crar01 opened the pull request:
Adding theme option, light and dark mode,
Refactor rename to controls.

crar01 committed:
Add theme option

Adding theme option, light and dark mode,
refactor rename to controls.

maxfie1d commented:
I can't wait that this change will be merged!

crar01 committed:
remove box border of list items

crar01 committed:
change colors for light mode

crar01 committed:
Screenshots dark and light

crar01 committed:
update Screenshots

- Update Screenshots
- Update Switcheroo portable

crar01 commented:
> I can't wait that this change will be merged!
Hi @maxfie1d : 
I have implemented the light and dark theme, you can see the screenshots here:
https://github.com/crar01/Switcheroo

milnak commented:
FYI, I integrated this change into [my fork](https://github.com/milnak/Switcheroo/)

crar01 committed:
update ui

- remove borders
- add cornerRadius
- add blurRadius

crar01 committed:
Upgrade projects to .NET Framework 4.8

Updated all project files to target .NET Framework 4.8 instead of 4.5. Increased ToolsVersion to 12.0 where applicable. Updated generated code attributes in designer files and changed app.config to reference the new framework version.

crar01 committed:
Add system theme support and auto-update on change

Introduces a 'System' option for theme selection, allowing the app to follow the OS light/dark mode. The theme updates automatically when the system preference changes by listening to UserPreferenceChanged events. Updates the theme logic and options UI to support this feature.

crar01 committed:
Update images and readme download section

Replaced dark.png and light.png with updated versions. Modified the readme to clarify the download options as light/dark/system.

crar01 committed:
Merge pull request #1 from crar01/updateUI

Update UI and add system theme

coezbek commented:
I also integrated this PR by @crar01 into my fork at https://github.com/coezbek/switcheroo

crar01 commented:
I published a new version (add system theme/rounded corners), you can download it from:
https://github.com/crar01/Switcheroo
